robomimic/models/base_nets.py

```diff
@@ -851,6 +851,11 @@ def __init__(
 
         # Get activation requested
         activation = CONV_ACTIVATIONS[activation]
+        
+        # Add layer kwargs
+        conv_kwargs["out_channels"] = out_channels
```

Comment:
should we assert these keys do not already exist in `conv_kwargs` or are they meant to be overwritten?

Reply:
Great question! If you see in the lines above, they can't actually exist already in `conv_kwargs` precisely because those kwargs are explicitly listed in the constructor (that's the reason for the error!)